### PR TITLE
chore(backend): upgrade to LSM Tigerbeetle version and tigerbeetle-node 0.11.11

### LINKED ISSUE #972: Make use of the LSM Tigerbeetle release for testcontainers (once ready)
- @sentientwaffle said there are no major demons anymore
- BUT: does not work on Apple M1 (however, docker works)
- TB and node client now in the same repo
- update TB together with node client

@koekiebox  remapped error codes that need to be cleaned up in `packages/backend/src/accounting/transfers.ts`
https://github.com/interledger/rafiki/pull/452/files#diff-e220292cea3c986078ff4fb24d3d3d9f4c949ae28ad3412026743f4e0aec6580

### LINKED ISSUE #1009: Update tigerbeetle testcontainer wait strategy
This message does not exist with version of TigerBeetle corresponding to the lastest version of `tigerbeetle-node`
https://github.com/interledger/rafiki/blob/dd07d2ff6f157c8b3fd671e9d74f7ea86e91ee5b/packages/backend/jest.setup.js#L98

https://github.com/tigerbeetledb/tigerbeetle/blob/main/src/tigerbeetle/main.zig

### PR DESCRIPTION
**Pull request by sabineschaller**
## Changes proposed in this pull request

- upgrade to tigerbeetle version sha256:621268959801084246679f25416a9451f553fe3697dd4f618f39503322079433
- upgrade to tigerbeetle-node 0.11.11
- remove old error codes
- increase test timeout
- remove gracefully shutting down individual tigerbeetle containers since that results in thread panic within gh actions (but added todo to re-enable that somehow)
  - #1094 1094

## Context

- fixes #972 
- fixes #1009 

## Checklist

- [ ] Related issues linked using `fixes #number`
- [ ] Tests added/updated
- [ ] Documentation added
- [ ] Make sure that all checks pass

### CODE REVIEW

**Comment by wilsonianb** [comment]
I think you can remove these to resolve (some) build errors
https://github.com/interledger/rafiki/blob/52b6bbfe32f37ed05a62ca9fe0de2bdf702d0530/packages/backend/src/accounting/errors.ts#L48-L51

**Comment by sabineschaller** [comment]
Tests are also failing because we are deleting postgres data in between tests but not tigerbeetle data. Tigerbeetle complains about duplicate liquidity and settlement accounts.

**Comment by wilsonianb** [comment]
> Tests are also failing because we are deleting postgres data in between tests but not tigerbeetle data. Tigerbeetle complains about duplicate liquidity and settlement accounts.

That's what all these are?
```
     /lib/x86_64-linux-gnu/libc.so: invalid ELF header
```

**Review by wilsonianb — COMMENTED** [review]
localenv is struggling for me
```
local-tigerbeetle-1 exited with code 1
local-tigerbeetle-1 exited with code 1
...
local-peer-backend-1     | {"level":30,"time":1675866191186,"pid":1,"hostname":"8977b51568a2","msg":"🐒 has 🚀. Get ready for 🍌🍌🍌🍌🍌"}
local-backend-1          | error(message_bus): error connecting to replica 0: error.Unexpected
local-peer-backend-1     | error(message_bus): error connecting to replica 0: error.Unexpected
local-tigerbeetle-1 exited with code 1
local-backend-1          | error(message_bus): error connecting to replica 0: error.Unexpected
local-peer-backend-1     | error(message_bus): error connecting to replica 0: error.Unexpected
local-backend-1          | error(message_bus): error connecting to replica 0: error.ConnectionRefused
local-tigerbeetle-1 exited with code 1
local-peer-backend-1     | error(message_bus): error connecting to replica 0: error.Unexpected
local-peer-backend-1     | error(message_bus): error connecting to replica 0: error.Unexpected
local-tigerbeetle-1 exited with code 1
...
```

File: packages/backend/src/accounting/tigerbeetle/errors.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { AccountId } from './utils'
 
 export class TigerbeetleCreateAccountError extends Error {
   constructor(public code: number) {
-    super(`CreateAccountError code: ${code}`)
+    super(`Tigerbeetle error code: ${code}`)
     this.name = 'CreateAccountError'
```
Comment (mkurapov) [inline packages/backend/src/accounting/tigerbeetle/errors.ts:10]: Optionally, we could also do `this.name = 'TigerbeetleCreateAccountError'`

File: packages/backend/jest.config.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ const packageName = require('./package.json').name
 module.exports = {
   ...baseConfig,
   clearMocks: true,
-  testTimeout: 15000,
+  testTimeout: 30000,
```
Comment (mkurapov) [inline packages/backend/jest.config.js:10]: I'm guessing once we have psql accounting by default, we can revert to 15000 and only set 30000 for the tigerbeetle tests?

**Review by mkurapov — APPROVED** [review]

File: packages/backend/jest.config.js (hunk 2 above)
Comment (sabineschaller) [reply packages/backend/jest.config.js:10]: I think so. I tried reverting already once I figured out where the problem was but 15 seconds wasn't enough.

**Review by wilsonianb — APPROVED** [review]
:+1: 
localenv looks good (had to cherry-pick :point_down:)
- https://github.com/interledger/rafiki/pull/1082

**Pull request merged**